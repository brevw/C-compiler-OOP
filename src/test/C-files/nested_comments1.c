// here we add a nested comment

/* comment
   /* nested comment */
   comment
*/
