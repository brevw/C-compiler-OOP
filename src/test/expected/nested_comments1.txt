IDENTIFIER(comment)
ASTERISK
DIV
Lexing: pass
